### PR TITLE
Edu

### PR DESCRIPTION
**Pull request by manjaha**

### CODE REVIEW

File: parse_xml.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import xml.etree.ElementTree as ET
+
```
Comment (ValentynTroian) [inline parse_xml.py]: It should be 2 empty lines before function according to the PEP8.

File: parse_xml.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+country_government = []
+countries = parse_and_remove('mondial-3.0.xml', 'country')
+for country in countries:
+    government = country.attrib['government']
+    name = country.attrib['name']
+    if ' ' in name and government not in country_government:
+        country_government.append(government.strip(' '))
+print(country_government)
```
Comment (ValentynTroian) [inline parse_xml.py]: I'd just add the main function on line 23:
```
def main():
    country_government = []
    ...
```
and then use it:
```
if __name__ == '__main__':
    main() 
```

**Comment by ValentynTroian** [comment]
Well done! No observation from my side.

**Comment by ValentynTroian** [comment]
I can't merge to the master due to "Conflicting files readcsv.py" issue

**Comment by manjaha** [comment]
:( what can I do to fix readcsv.py and finally merge to the master?

**Comment by ValentynTroian** [comment]
Merged :)

**Pull request closed without merging**